doc/nrf/gs_installing.rst

```diff
@@ -535,20 +514,17 @@ If you chose to use |SES| for :ref:`building and programming a sample applicatio
 
    a. Navigate to the :file:`bin` directory.
    #. Run the :file:`emStudio` executable file.
-   #. Select :guilabel:`File` > :guilabel:`Open nRF Connect SDK Project`.
+   #. Click :guilabel:`OK` on the pop-up window that opens.
+   #. Set the GNU ARM Embedded Toolchain directory to :file:`c:/gnuarmemb`.
+   #. Close the :guilabel:`nRF Connect options` window.
 
-      .. figure:: images/ses_open.png
-         :alt: Open nRF Connect SDK Project menu
+      .. figure:: images/ses_GNU_notset.png
+         :alt: Open GNU ARM Embedded Toolchain directory
 
-         Open nRF Connect SDK Project menu
+         Open GNU ARM Embedded Toolchain directory
```

Comment:
I believe "the" should be added to line 524, so that it reads, " Open the GNU ARM Embedded Toolchain directory" .